src/main/java/internetshop/controller/GetPurchaseHistoryController.java

```diff
@@ -0,0 +1,26 @@
+package internetshop.controller;
+
+import internetshop.annotation.Inject;
+import internetshop.model.Order;
+import internetshop.service.UserService;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetPurchaseHistoryController extends HttpServlet {
```

Comment:
maybe better GetAllOrdersByUserController? or something else? without history:)

Reply:
in user i alredy have field "OrderHistory", so that contoller just getting that list